assets/Script/UI/PlayerTradePage.ts

```diff
@@ -556,13 +566,18 @@ export function PlayerTradePage(): m.Comp<{
                                         return false;
                                     }
                                     const res = isResourceFilterEmpty ? true : resourceFilter[trade.resource];
-                                    let p = true;
-                                    if (minPriceFilter > 0) {
+                                    let p = true;                                    
+                                    if ( maxResourceAmount != NaN && maxResourceAmount > 0 ) {
```

Comment:
Maybe use `isFinite(xxx)` instead of `xxx != NaN`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite

Reply:
Changed to use isFinite